### ISSUE TITLE
Pass through params like {equal:compare.structural} to the computed() within a createTransformer

### ISSUE THREAD

**Issue by liron00** [mobxjs/mobx-utils#166]
I often need a transformer that compares outputs structurally.

Generalizing #129, the transformer uses computed() and it would be nice to pass through any of the normal set of computed options.

**Comment by mweststrate** [comment]
Sounds useful to be able to provide a limited set of the available, relevant computed options to `createTransformer`!

**Comment by ItamarShDev** [comment]
@liron00 Will be happy to get a reproduction code to be able to see the case clearly
Thank you

**Comment by liron00** [comment]
@ItamarShDev I don't have a specific syntax in mind that I'd like to use, but you may be interested to see how my workaround works to enable a transformer with structural equality:
```
import {createTransformer as mobxCreateTransformer} from 'mobx-utils'

function createTransformer(func, options) {
  // Structural comparison
  const mobxTransformer = mobxCreateTransformer(
    x => x && JSON.stringify(func(x)),
  )
  return x => {
    const jsonResult = mobxTransformer(x)
    return jsonResult && JSON.parse(jsonResult)
  }
}

**Comment by mweststrate** [comment]
Should be fixed in 5.5.3
